Makefile

```diff
@@ -143,11 +143,15 @@ ADDONS_BINDING_GYPS := \
 	$(filter-out test/addons/??_*/binding.gyp, \
 		$(wildcard test/addons/*/binding.gyp))
 
+ADDONS_BINDING_SOURCES := \
+	$(filter-out test/addons/??_*/*.cc, $(wildcard test/addons/*/*.cc)) \
+	$(filter-out test/addons/??_*/*.h, $(wildcard test/addons/*/*.h)) \
```

Comment:
I tend to write it like that because it keeps the diff small when you add lines later on.  But you're right, the rest of the Makefile doesn't use that style.